pandas/_libs/tslibs/period.pyx

```diff
@@ -1241,6 +1241,24 @@ cdef class _Period(object):
 
     @property
     def hour(self):
+        """
+        Get hours of a day that a Period falls on.
+
+        Returns
+        -------
+        int
```

Comment:
For the explanation note that it's betwene 0 and 23.

```
int
    The hour as an integer, between 0 and 23.
```

Reply:
sure